ui/src/components/Filter/Filter.tsx

```diff
@@ -21,11 +31,31 @@ interface FilterList {
   header: string;
 }
 
+/**
+ * To convert the first letter of the input word to Upper Case
+ */
+export const titleCase = (filterName: string) => {
```

Comment:
I think we should move this function to `utils` directory so that reusing it becomes more easy 
cc @sthaha